docs/stability.md

```diff
@@ -29,6 +29,7 @@ Things considered unstable for 2.x:
 * Service discovery integrations, with the exception of `static_configs` and `file_sd_configs`
 * Go APIs of packages that are part of the server
 * HTML generated by the web UI
+* The /api/v1/notifications and /api/v1/notifications/live endpoints.
```

Comment:
There are a bunch more API endpoints that are marked as experimental on https://prometheus.io/docs/prometheus/latest/querying/api/#http-api (querying exemplars, target metadata, metric metadata, and in the not-yet-published 3.0 version also the endpoint for parsing a PromQL query into a tree). I would maybe just leave it up to that page to document the specific stability and note that above by saying "...unless specifically marked as experimental."